Source/JavaScriptCore/jsc.cpp

```diff
@@ -3970,7 +3972,19 @@ void CommandLine::parseArguments(int argc, char** argv)
         }
         if (!strcmp(arg, "-s")) {
 #if OS(UNIX)
-            initializeSignalHandling();
+            uint32_t key = 0;
+            int mask = 0;
+#if HAVE(MACH_EXCEPTIONS)
+            mask |= toMachMask(Signal::IllegalInstruction);
+            mask |= toMachMask(Signal::AccessFault);
+            mask |= toMachMask(Signal::FloatingPoint);
+            mask |= toMachMask(Signal::Breakpoint);
+#if !OS(DARWIN)
+            mask |= toMachMask(Signal::Abort);
+#endif // !OS(DARWIN)
```

Comment:
Does HAVE(MACH_EXCEPTIONS) imply OS(DARWIN)? Perhaps this can be removed?

Reply:
I think in theory there could be BSD distro's that have Mach. We can just leave it for now.